tempesta_fw/t/functional/tests/helpers/apache.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+__author__ = 'Tempesta Technologies Inc.'
+__copyright__ = 'Copyright (C) 2016 Tempesta Technologies. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+import subprocess
+
+
+def is_apache():
+	try:
+		dist = get_dist()
+		if dist == "debian":
+			p = subprocess.Popen(["apache2", "-v"], 
+					     stdout=subprocess.PIPE)
+		elif dist == "centos":
+			p = subprocess.Popen(["httpd", "-v"], 
+					     stdout=subprocess.PIPE)
```

Comment:
# tempesta_fw/t/functional/tests.py test_cache
         Traceback (most recent call last):
           File "tempesta_fw/t/functional/tests.py", line 44, in <module>
             print("test:{}".format(tclass().get_name()))
           File "/root/tempesta/tempesta_fw/t/functional/tests/test_cache.py", line 30, in __init__
             self.apache_cfg = conf.ApacheConfig()
           File "/root/tempesta/tempesta_fw/t/functional/tests/helpers/conf.py", line 79, in __init__
             Config.__init__(self, '/etc/apache2/apache2.conf', new=False)
           File "/root/tempesta/tempesta_fw/t/functional/tests/helpers/conf.py", line 44, in __init__
             open(self.name, "a+")
         IOError: [Errno 2] No such file or directory: '/etc/apache2/apache2.conf'

This is CentOS 7:

        # grep '\<ID\>' /etc/os-release 
        ID="centos"

 Why apache2.conf is searched?